### PR TITLE
Added Escalation Type

### PR DESCRIPTION
**Pull request by harikac1**

### CODE REVIEW

File: pkg/connector/escalations.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	return rv, "", nil, nil
+}
+
+func (o *escalationResourceType) Entitlements(ctx context.Context, resource *v2.Resource, _ *pagination.Token) ([]*v2.Entitlement, string, annotations.Annotations, error) {
+	var rv []*v2.Entitlement
+
+	assignmentOptions := []ent.EntitlementOption{
+		ent.WithGrantableTo(resourceTypeEscalation),
+		ent.WithDisplayName(fmt.Sprintf("%s Escalation Member", resource.DisplayName)),
+		ent.WithDescription(fmt.Sprintf("Is member of the %s escalation in Opsgenie", resource.DisplayName)),
```
Comment (loganintech) [inline pkg/connector/escalations.go:83]: I'd probably word this different. You're not exactly a "member" of an escalation. Escalation policies apply to your team

**Comment by ggreer** [comment]
Besides the git conflict, is there anything blocking this? It'd be nice to get it merged.

**Comment by harikac1** [comment]
> Besides the git conflict, is there anything blocking this? It'd be nice to get it merged.

So, looking more into Escalations they're basically rules to notify on-call people. They don't fit into resources/entitlements I think. Not sure how do we want to proceed.

**Comment by mgaeta** [comment]
Hey @harikac1, are you still working on this PR?

**Comment by mgaeta** [comment]
@harikac1 I'm going to close this PR on November 1.

**Comment by harikac1** [comment]
> @harikac1 I'm going to close this PR on November 1.

@mgaeta sorry, I thought I responded. We can close this

**Pull request closed without merging**